README.md

```diff
@@ -23,14 +23,25 @@ It does not make a copy. Make sure your code is checked into a source control
 repository like Git and that you have no outstanding changes to commit before
 running this tool.
 
+```sh
+$ cd my-ember-app-or-addon
+$ npx ember-angle-brackets-codemod ./path/of/files/ or ./some**/*glob.hbs
```

Comment:
We should ask for `--no-telemetry` to explicitly opt-out of telemetry gathering.

Reply:
That makes sense given that you want telemetry to be the default use case.

However, personally I like the current choice to have `--telemetry=` replace `--url=`. To me if you don't provide `--telemetry=`, then `--no-telemetry` is not needed unless you wanted to use a default URL for telemetry - is that what you are suggesting?

Reply:
I agree with @tomwayson, also having too many options/flags, for one thing, seems to bring in extra complexity

Reply:
When using `yargs` you get `--no-telemetry` "for free" when you configure `--telemetry` to mean something. I think it is perfectly fine for the implementation to not require `--telemetry` or `--no-telemetry` (e.g. assume `--no-telemetry` if there is no `--telemetry` value provided) but our docs **should** explicitly pass the argument.